components/EditCarForm/index.js

```diff
@@ -30,75 +54,42 @@ const groups = [
   },
 ];
 
-export default function EditCarForm({ activeCar, onSubmit, form }) {
-  if (form === "edit") {
-    return (
-      <form onSubmit={onSubmit}>
-        {groups.map((group) => {
-          return (
-            <fieldset key={group.id}>
-              <p>{group.headline}</p>
-              {Object.keys(activeCar).map((attribute, index) => {
-                if (group.content.includes(attribute)) {
-                  // const attributeValue = activeCar[attribute];
-                  // const attributeValue = initialCars[index][attribute];
-
-                  const type = "text";
-                  // typeof attributeValue === "string" ? "text" : "number";
-                  return (
-                    <div key={group.id + index}>
-                      <label>
-                        {attribute}:
-                        <input
-                          name={attribute}
-                          type={type}
-                          defaultValue={activeCar[attribute]}
-                        ></input>
-                      </label>
-                    </div>
-                  );
-                } else {
-                  return null;
-                }
-              })}
-            </fieldset>
-          );
-        })}
+export default function EditCarForm({ onSubmit, form, activeCar }) {
+  return (
+    <form onSubmit={onSubmit}>
+      {groups.map((group) => {
+        return (
+          <fieldset key={group.id}>
+            <p>{group.headline}</p>
```

Comment:
Thank you for reviewing 😄 This paragraph should show the name of the group of attributes in the fieldset. I renamed it to"description",  the old name could be confusing. The h element is not the best way in my opinion, because i need that multiple times in the same way, I can't use h1 -> h2 -> h3 because that would be misleading. And I also can't use the same h element multiple times.